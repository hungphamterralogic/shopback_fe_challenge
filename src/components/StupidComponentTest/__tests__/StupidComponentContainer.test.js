import React from 'react';

import { genInitialMockStore, renderWithStore } from '../../../test/reduxComponentTestHelpers';

import StupidComponentContainer from '../StupidComponentTestContainer';

describe('StupidComponentContainer', () => {
  test('def', () => {
    const store = genInitialMockStore({
      'stores.message': 'hi there'
    });
    expect(renderWithStore(<StupidComponentContainer />, store)).toMatchSnapshot();
  });
});
