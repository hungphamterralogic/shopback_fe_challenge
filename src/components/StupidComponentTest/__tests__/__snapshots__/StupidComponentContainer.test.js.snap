// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StupidComponentContainer def 1`] = `
<p>
  hi there
</p>
`;
